src/components/RichTextEditor/Plugins/helper.tsx

```diff
@@ -25,3 +27,108 @@ export const ButtonGroupWrapper = ({ index, children }: ButtonGroupProps) => (
 export const ButtonWrapper = ({ children, id }: { children: ReactNode; id: string }) => (
     <div data-plugin-id={id}>{children}</div>
 );
+
```

Comment:
This helpers is to be functions used by any plugin. I guess this methods below are only for the TextStyle Plugin. So I would suggest because it is so clothed to move them to the plugin that use them. Also names indicates that, so it is not right place for them to be!

Reply:
Moved `ButtonGroupWrapper` to the toolbar where it belongs to and also the `getButtonStyleCssProperties` to the buttonPlugin, all other functions are are used in multiple plugins + serializer... 

The `getButtonStyleCssProperties` could be moved to the button plugin

Reply:
What about `getLiStyles` would that not go into ListPlugin? It indicates the name that.

Reply:
Also moved without mentioning 💯

Reply:
Also fixed the TextStylePlugin without styles (not displayed in the toolbar anymore)